cmake/VMCRequiredPackages.cmake

```diff
@@ -0,0 +1,27 @@
+#------------------------------------------------
+# The Geant4 Virtual Monte Carlo package
+# Copyright (C) 2014 Ivana Hrivnacova
+# All rights reserved.
+#
+# For the licensing terms see geant4_vmc/LICENSE.
+# Contact: root-vmc@cern.ch
+#-------------------------------------------------
+
+# Configuration file for CMake build for VMC packages
+# which finds all required and optional packages.
+#
+# I. Hrivnacova, 07/12/2021
+
+#message(STATUS Processing Geant4VMCRequiredPackages)
+
+#-- ROOT (required) ------------------------------------------------------------
+find_package(ROOT CONFIG COMPONENTS Geom EG REQUIRED)
```

Comment:
Just for my curiosity, do you remember the rationale behind using `CONFIG` here explicitely as opposed to just being implicit about the lookup method, such as `find_package(ROOT COMPONENTS Geom EG REQUIRED)`?

Reply:
As far as I can remember we did it to be explicit. Honestly I don't remember that there was another reason @ihrivnac ?

Reply:
Cmake documentation (https://cmake.org/cmake/help/v3.22/command/find_package.html) says:
"The CONFIG option, the synonymous NO_MODULE option, or the use of options not specified in the basic signature all enforce pure Config mode. In pure Config mode, the command skips Module mode search and proceeds at once with Config mode search."

If I remember well, we added these to require finding the packages via Config file when moving to CMake build in VMC; also in case of Root this prevents from using ROOT built with the old build system, which may not provide the targets as we use them in our build.

Reply:
Thank you for your answers!